editor/spatial_editor_gizmos.cpp

```diff
@@ -1726,9 +1728,69 @@ void SkeletonSpatialGizmoPlugin::redraw(EditorSpatialGizmo *p_gizmo) {
 	}
 
 	Ref<ArrayMesh> m = surface_tool->commit();
+	p_gizmo->add_handles(handles, get_material("handles"));
+	p_gizmo->add_collision_triangles(m);
+	p_gizmo->add_collision_segments(handles);
 	p_gizmo->add_mesh(m, false, skel->get_skeleton());
 }
 
+String SkeletonSpatialGizmoPlugin::get_handle_name(const EditorSpatialGizmo *p_gizmo, int p_idx) const {
```

Comment:
It uses the handle code to generate the "indicators", but clicking on them fail.

I think it's because the parent code is override the indicator's collision but I couldn't find out why.

Reply:
So I documented the existing behaviour.

Reply:
We discussed and the consensus is to remove the extra code until the gizmo system is improved to support full transforms for handles.